src/content/add-to-app/ios/project-setup.md

```diff
@@ -5,75 +5,61 @@ description: Learn how to integrate a Flutter module into your existing iOS proj
 ---
 
 Flutter UI components can be incrementally added into your existing iOS
-application as embedded frameworks. There are a few ways to embed Flutter 
-in your existing application.
+application as embedded frameworks.
+To embed Flutter in your existing application,
+consider one of the following three methods.
 
-1. **Use the CocoaPods dependency manager and installed Flutter SDK.**
-  In this case, the `flutter_module` is compiled from
-  the source each time the app is built. (Recommended.)
+| Embedding Method | Methodology | Benefit |
+|---|---|---|
+| Use CocoaPods _(Recommended)_ | Install and use the Flutter SDK and CocoaPods. Flutter compiles the `flutter_module` from source each time Xcode builds the iOS app. | Least complicated method to embed Flutter into your app. |
+| Use [iOS frameworks][] | Create iOS frameworks for Flutter components, embed them into your iOS, and update your existing app's build settings. | Doesn't require every developer to install the Flutter SDK and Cocoapods on their local machines. |
+| Use iOS frameworks and CocoaPods | Embed the frameworks for your iOS app and the plugins in Xcode, but distribute the Flutter engine as a CocoaPods podspec. | Provides an alternative to distributing the large Flutter engine (`Flutter.xcframework`) library. |
 
-1. **Create frameworks for the Flutter engine, your compiled Dart code,
-  and all Flutter plugins.** Here, you manually embed the frameworks,
-  and update your existing application's build settings in Xcode.
-  This can be useful for teams that don't want to require every developer
-  to have the Flutter SDK and Cocoapods installed locally.
+{:.table .table-striped}
 
-1. **Create frameworks for your compiled Dart code,
-  and all Flutter plugins. Use CocoaPods for the Flutter engine.** 
-  With this option, embed the frameworks for your application
-  and the plugins in Xcode, but distribute the
-  Flutter engine as a CocoaPods podspec.
-  This is similar to the second option, but it provides
-  an alternative to distributing the large Flutter.xcframework.
+[iOS frameworks]: {{site.apple-dev}}/library/archive/documentation/MacOSX/Conceptual/BPFrameworks/Concepts/WhatAreFrameworks.html
 
-For examples using an app built with UIKit, 
-see the iOS directories in the [add_to_app code samples][]. 
-For an example using SwiftUI, see the iOS directory in [News Feed App][].
+Using Flutter modules [increases your app size][].
```

Comment:
We haven't introduced Flutter modules yet. Should we use simpler language here? Perhaps:

```suggestion
Adding Flutter to your existing iOS app [increases its size][].
```